media-translation/cloud-client/translate_from_mic.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Cloud Media Translation API sample application using a microphone.
+
+Example usage:
+    python translate_from_mic.py
+"""
+
+# [START media_translation_translate_from_mic]
+from __future__ import division
+
+import itertools
+
+from google.cloud import mediatranslation
+import pyaudio
+from six.moves import queue
```

Comment:
Does `six` need to be added to `requirments.txt`?
https://pypi.org/project/six/

Reply:
Done.